### PR TITLE
Minor improvement in the node types explanation

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.3+ |
| Fixed tickets | - |

### CODE REVIEW

**Comment by timglabisch** [comment]
:+1:

File: components/config/definition.rst [hunk 1]
```diff
@@ -95,13 +95,13 @@ Node Type
...
-node definition. Node type are available for:
+node definition. Node types are available for:
 
-* scalar
+* scalar (generic type that includes booleans, strings, integers, floats and ``null``)
 * boolean
 * integer (new in 2.2)
 * float (new in 2.2)
-* enum (new in 2.1)
+* enum (new in 2.1) (similar to array, but it only allows a finite set of values)
```
Comment (stof) [inline components/config/definition.rst]: it is similar to scalar, not to array

Comment (javiereguiluz) [inline components/config/definition.rst]: Fixed. Thanks.

**Comment by javiereguiluz** [comment]
The errors reported by reviewers have been addressed and there are no new comments. I think we can safely tag this PR as *finished*.

**Comment by weaverryan** [comment]
Thanks Javier!

**Pull request merged**